database.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$connection = mysqli_connect("localhost", "root", "", "books_db");
+
+if (!$connection) {
+    die("Failed to connect to database: " . mysqli_connect_error());
```

Comment:
Consider creating a `Database` class to manage the connection consistently and employing dependency injection to enhance scalability and code maintenance.